src/components/header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+const HeaderContainer = styled.div`
+  background-color: ${({ theme }) => theme.colors.light};
+  padding: ${({ theme }) => theme.sizes.spacing.lg};
+  display: flex;
+  justify-content: space-between;
+  h1 {
+    font-weight: lighter;
+    font-size: ${({ theme }) => theme.sizes.fonts.xl};
+  }
+  .fa-bars {
+    cursor: pointer;
+  }
+  .fa-bag-shopping:hover,
+  .fa-magnifying-glass:hover,
+  .fa-bars:hover {
+    filter: opacity(0.7);
+  }
+  .fa-magnifying-glass {
+    cursor: pointer;
+    margin-left: 5px;
```

Comment:
grab a margin from the theme

Reply:
Done! - need to get used to using the theme, Thanks